scm/github.go

```diff
@@ -430,8 +246,7 @@ func (s *GithubSCM) DemoteTeacherToStudent(ctx context.Context, opt *UpdateEnrol
 	if !opt.valid() {
 		return fmt.Errorf("missing fields: %+v", opt)
 	}
-	role := OrgMember
-	_, _, err := s.client.Organizations.EditOrgMembership(ctx, opt.User, opt.Organization, &github.Membership{Role: &role})
+	_, _, err := s.client.Organizations.EditOrgMembership(ctx, opt.User, opt.Organization, &github.Membership{Role: github.String(OrgMember)})
```

Comment:
We could create vars for memberships similar to the ones for push and pull access in helpers.go

Reply:
This is coming in #1114.